documentation/content/docs/02-template-syntax/05-element-directives.md

```diff
@@ -407,53 +407,7 @@ action = (node: HTMLElement, parameters: any) => {
 }
 ```
 
-Actions are functions that are called when an element is created. They can return an object with a `destroy` method that is called after the element is unmounted:
-
-```svelte
-<!--- file: App.svelte --->
-<script>
-	/** @type {import('svelte/action').Action}  */
-	function foo(node) {
-		// the node has been mounted in the DOM
-
-		return {
-			destroy() {
-				// the node has been removed from the DOM
-			}
-		};
-	}
-</script>
-
-<div use:foo />
-```
-
-An action can have a parameter. If the returned value has an `update` method, it will be called whenever that parameter changes, immediately after Svelte has applied updates to the markup.
-
-> Don't worry about the fact that we're redeclaring the `foo` function for every component instance — Svelte will hoist any functions that don't depend on local state out of the component definition.
-
-```svelte
-<!--- file: App.svelte --->
-<script>
-	export let bar;
-
-	/** @type {import('svelte/action').Action}  */
-	function foo(node, bar) {
-		// the node has been mounted in the DOM
-
-		return {
-			update(bar) {
-				// the value of `bar` has changed
-			},
-
-			destroy() {
-				// the node has been removed from the DOM
-			}
-		};
-	}
-</script>
-
-<div use:foo={bar} />
-```
+Read more in [svelte/action](/docs/svelte-action) page.
```

Comment:
Don't know how I feel about this. `transition` for example also has a dedicated page but a short description in here. It's not strictly the same, but having a sub headline only to redirect feels wrong. Could we have a short summaray here and then a link as a compromise?

Reply:
Added it back